bonsai/Modules/NewOperation/NewOperationView.swift

```diff
@@ -116,7 +117,17 @@ struct NewOperationView: View {
          .navigationBarTitleDisplayMode(.large)
          .toolbar {
             ToolbarItem(placement: .navigationBarLeading) {
-               NavigationBackButton(isPresented: $isPresented)
+               Button(action: {
+                  confirmationPresented = true
+               }) {
+                  Text("Cancel")
+                       .foregroundColor(BonsaiColor.secondary)
+               }
+               .confirmationDialog("Are you sure?", isPresented: $confirmationPresented, actions: {
```

Comment:
I think we should mark String text with some TO DO comments to add localization later

Reply:
Fixed